packages/api/src/add-build.js

```diff
@@ -18,8 +18,8 @@
 * and builds
 * returns analytics object and adds such data to database
 */
-const { Firestore } = require('@google-cloud/firestore');
 const { buildPassingPercent, TestCaseAnalytics } = require('../lib/analytics.js');
+const Firestore = require('@google-cloud/firestore');
```

Comment:
Are these things doing the same thing? I was copying [this](https://github.com/googleapis/nodejs-firestore), but that certainly looks cleaner.

Reply:
As far as I can tell they are the same, the brackets are useful for when more than one variable is set by the requirement.